lib/online_order.rb

```diff
@@ -0,0 +1,90 @@
+require "csv"
+require "ap"
+require_relative "order.rb"
+require_relative "customer.rb"
+
+module Grocery
+  class OnlineOrder < Order
+    attr_accessor :customer, :online_order_id, :products, :fulfillment_status
+
+      @customer = Customer.find(id)
+      @products = products
+      @online_order_id = online_order_id
+      @fulfillment_status = fulfillment_status
+
+    end
+
+    def self.all
+      online_orders_array = CSV.read("support/online_orders.csv", "r")
+      online_orders = []
+
+      online_orders_array.each do |order|
+        order_id = order[0].to_i
+        products_hash = {}
+        id = order[2].to_i
+        status = order[3].to_sym
```

Comment:
It might be wise to call the variable on line 24 `customer_id`, to make it very clear what it is.

Reply:
This comment applies to pretty much anywhere you're using the variable `id` in this file.